2023-09-19 12:16:20,141 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-19 12:16:20,142 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:353]
2023-09-19 12:16:28,179 INFO: Video processing has finished [in /Users/p/Documents/Github/surveil/video_app/utils.py:362]
2023-09-19 12:16:28,179 INFO: csv successfully written locally [in /Users/p/Documents/Github/surveil/video_app/utils.py:384]
2023-09-19 12:16:28,346 INFO: Class Detections method write_images() has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:370]
2023-09-19 12:16:28,961 INFO: images successfully written locally [in /Users/p/Documents/Github/surveil/video_app/utils.py:374]
2023-09-19 12:16:31,958 INFO: images successfully written to s3 bucket [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:168]
2023-09-19 12:16:37,136 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection5_person.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,136 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection2_car.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,137 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection6_person.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,138 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection1_person.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,138 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection3_train.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,139 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/images/detection4_person.jpg [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,139 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/csvs/data_20230919_121628.csv [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-19 12:16:37,141 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/videos/VIRAT_000001-0210-0220.mp4 [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-20 15:01:43,089 INFO: Video App startup [in /Users/p/Documents/Github/surveil/video_app/__init__.py:40]
2023-09-20 15:03:16,371 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-20 15:03:16,399 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:353]
2023-09-20 15:03:34,964 INFO: Video processing has finished [in /Users/p/Documents/Github/surveil/video_app/utils.py:362]
2023-09-20 15:03:34,965 INFO: csv successfully written locally [in /Users/p/Documents/Github/surveil/video_app/utils.py:384]
2023-09-20 15:03:40,509 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/csvs/data_20230920_150334.csv [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-20 15:05:47,716 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-20 15:05:47,717 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:353]
2023-09-20 15:06:08,234 INFO: Video processing has finished [in /Users/p/Documents/Github/surveil/video_app/utils.py:362]
2023-09-20 15:06:08,235 INFO: csv successfully written locally [in /Users/p/Documents/Github/surveil/video_app/utils.py:384]
2023-09-20 15:06:13,828 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/csvs/data_20230920_150608.csv [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-20 15:13:42,013 INFO: Video App startup [in /Users/p/Documents/Github/surveil/video_app/__init__.py:40]
2023-09-20 15:14:01,322 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-20 15:14:01,339 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:353]
2023-09-20 15:14:20,075 INFO: Video processing has finished [in /Users/p/Documents/Github/surveil/video_app/utils.py:362]
2023-09-20 15:14:20,076 INFO: csv successfully written locally [in /Users/p/Documents/Github/surveil/video_app/utils.py:384]
2023-09-20 15:14:25,448 INFO: Deleted file /Users/p/Documents/Github/surveil/temp_data/csvs/data_20230920_151420.csv [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:199]
2023-09-20 15:22:39,097 INFO: Video App startup [in /Users/p/Documents/Github/surveil/video_app/__init__.py:40]
2023-09-20 15:22:48,756 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-20 15:22:48,773 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:359]
2023-09-20 15:22:49,886 ERROR: Exception on /processing [GET] [in /Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/p/Documents/Github/surveil/video_app/routes/main_routes.py", line 139, in processing
    detectionsInVideo.process_video(model, VIDEO, CLASSES, YOUTUBE)
  File "/Users/p/Documents/Github/surveil/video_app/utils.py", line 360, in process_video
    self.results = list(model.track(source=VIDEO, conf=0.5, iou=0.5,
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/engine/model.py", line 277, in track
    return self.predict(source=source, stream=stream, **kwargs)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/engine/model.py", line 254, in predict
    return self.predictor.predict_cli(source=source) if is_cli else self.predictor(source=source, stream=stream)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/engine/predictor.py", line 195, in __call__
    return list(self.stream_inference(source, model, *args, **kwargs))  # merge list of Result into one
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 35, in generator_context
    response = gen.send(None)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/engine/predictor.py", line 227, in stream_inference
    self.setup_source(source if source is not None else self.args.source)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/engine/predictor.py", line 208, in setup_source
    self.dataset = load_inference_source(source=source, imgsz=self.imgsz, vid_stride=self.args.vid_stride)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/data/build.py", line 159, in load_inference_source
    dataset = LoadStreams(source, imgsz=imgsz, vid_stride=vid_stride)
  File "/Users/p/miniconda3/envs/video-metrics/lib/python3.10/site-packages/ultralytics/data/loaders.py", line 56, in __init__
    raise ConnectionError(f'{st}Failed to open {s}')
ConnectionError: 1/1: https://www.youtube.com/watch?v=X6-Nt-hxeL4&ab_channel=Nativebirds... Failed to open https://rr5---sn-a5msener.googlevideo.com/videoplayback?expire=1695270169&ei=uXALZaqlE6eR2_gPh-O1kAs&ip=2600%3A1700%3A4e02%3A310%3A2963%3A8cc%3Aac13%3Aa9e&id=o-AMd6Kf3-koGsF_d2U6x9Gb-RAB_vzz7f2_0ZKDzEA03U&itag=22&source=youtube&requiressl=yes&mh=zb&mm=31%2C26&mn=sn-a5msener%2Csn-o097znsz&ms=au%2Conr&mv=m&mvi=5&pl=47&pcm2=no&initcwndbps=992500&spc=UWF9f0pCjJTdH2x36Vu1ktargCBCKUEis1pA1q6TEg&vprv=1&svpuc=1&mime=video%2Fmp4&ns=gZoNU_PnMQCxUCXFoBqyYloP&cnr=14&ratebypass=yes&dur=9.613&lmt=1695248117088764&mt=1695247781&fvip=4&fexp=24007246&beids=24350017&c=WEB&txp=5318224&n=bToqoxiFaFlGz72&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cpcm2%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Ccnr%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAJcppWpj88F_w3VKldvqK8RwULZQXI56OBVfQ8bfiwakAiAlL1TcQT6hJXJYkRDcz8h6v40x7UOeyQgC1lfQoFdJlA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgVZ1F7DqtqqT8zlz9eFGTkwiORL5W2pXu78MugeWzR_MCIQCtevDC15hYNiG-02TKNbYmRZOJJmPaqonaqdCersfqSw%3D%3D
2023-09-20 15:23:24,985 INFO: Video App startup [in /Users/p/Documents/Github/surveil/video_app/__init__.py:40]
2023-09-20 15:23:28,472 INFO: Processing route was hit! [in /Users/p/Documents/Github/surveil/video_app/routes/main_routes.py:120]
2023-09-20 15:23:28,489 INFO: Video processing has begun [in /Users/p/Documents/Github/surveil/video_app/utils.py:359]
